src/index.js

```diff
@@ -0,0 +1,255 @@
+import './index.css';
+import commentCounter from './comment-counter.js';
+import itemCounter from './item-counter.js';
+import Image from './logo.png';
+
+let season;
+let number;
+
+const getMovies = async () => {
+  const response = await fetch(' https://api.tvmaze.com/seasons/1/episodes');
```

Comment:
- [ ] I observed that you repeated the `api url` severally, to follow the DRY principle; kindly create a variable for it and assess it as applicable. **Note: The same thing is applicable to the involvement `api` as well. Thanks.**

Reply:
Sure , we got it and it is a good idea

Reply:
@touredri Great!

Reply:
It is adjusted as instructed to apply DRY